specs/include_can_process_strings_from_pipeline_should_pass.ps1

```diff
@@ -0,0 +1,18 @@
+[string[]]$files = @("TestFile1.ps1","TestFile2.ps1");
+
+Out-File -FilePath "TestFile1.ps1" -InputObject "function Test-Function1 { return 1; }";
+Out-File -FilePath "TestFile2.ps1" -InputObject "function Test-Function2 { return 2; }";
+
+$files | Include;
+
+Task default -depends Test;
+
+BuildTearDown {
+    @("TestFile1.ps1","TestFile2.ps1","Function:\Test-Function1","Function:\Test-Function2") | ForEach-Object {Remove-Item $_;} | Out-Null;
+}
+
+Task Test {
+    Assert ($(Test-Path "Function:\Test-Function1") -ne $null) "Test-Function1 is not accessible";
```

Comment:
`Test-Path` will return `$true/$false` so no need for the $null check. 

If `Test-Path` returns `$false`, you're checking if that doesn't equal `$null`, which it doesn't, so the assert passes when it shouldn't if the function doesn't actually exist.

You can test this by putting in a bogus function name like so:

```powershell
Assert (Test-Path 'Function:\BogusFunction') 'BogusFunction is not accessible'
```

```powershell
Assert ($(Test-Path 'Function:\BogusFunction') -ne $null) 'BogusFunction  is not accessible'
```



 `Assert (Test-Path ...)` will be fine.

Reply:
Yeah... not sure what I was thinking when I wrote that. I'm guessing I intended to write "-ne $false". Should have tested my test.

Thanks; I'll get it corrected.

Reply:
Pushed the fix now :)